install.sh

```diff
@@ -295,14 +269,17 @@ echo -e "${RESET}"
 echo_info "Updating crew package information...\n"
 # Without setting LD_LIBRARY_PATH, the mandb postinstall fails
 # from not being able to find the gdbm library.
-export LD_LIBRARY_PATH=$(crew const CREW_LIB_PREFIX | sed -e 's:CREW_LIB_PREFIX=::g')
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CREW_PREFIX/lib:$CREW_PREFIX/lib64
 # Since we just ran git, just update package compatibility information.
 crew update compatible
 
+echo_info "Installing additional bootstrap packages...\n"
+yes | crew install musl_zstd rsync pixz
```

Comment:
Are these packages necessary for a minimal install?

Reply:
We need musl_zstd to install everything with zstd compression (which is most of the core packages), same deal with pixz for xz packages, and crew complains if we don't have rsync installed.